File: fizzbuzz.c
#include <stdio.h>

int main() {
	for (int i = 0; i < 100; i++) {
		int d = 0;
		if (i % 3 == 0) {
			printf("Fizz");
		} else {
			d++;
		}
		if (i % 5 == 0) {
			printf("Buzz");
		} else {
			d++;
		}

		if (d == 2) {
			printf("%d", i);
		}
		printf("\n");
	}
	return 0;
}

File: fizzbuzz.go
package main

import "fmt"

func main () {
	for i := 0; i < 100; i++ {
		var p int;
		if i % 3 == 0 {
			fmt.Print("Fizz")
			p =+ 1
		}
		if i % 5 == 0 {
			fmt.Print("Buzz")
			p += 1
		}
		if p == 0 { fmt.Printf("%d", i) }
		fmt.Println("")
	}
}
